src/renderer/api/endpoints/nodes.api.ts

```diff
@@ -56,39 +56,53 @@ export class Node extends KubeObject {
   static apiBase = "/api/v1/nodes";
 
   spec: {
-    podCIDR: string;
-    externalID: string;
+    podCIDR?: string;
+    podCIDRs?: string[];
+    providerID?: string;
+    externalID: string; // Deprecated. see: https://issues.k8s.io/61966
```

Comment:
If it deprecated then it should be marked as optional and maybe even add a comment.
```suggestion
    /**
     * @deprecated see https://issues.k8s.io/61966
     */
    externalID?: string;
```